.github/workflows/android_tests.yml

```diff
@@ -0,0 +1,43 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Android tests
+
+# Controls when the action will run. 
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    runs-on: macos-latest
+    timeout-minutes: 30
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 12
+
+      -  name: Android Emulator Runner
+         uses: ReactiveCircus/android-emulator-runner@v2.13.0
+         with:
+           api-level: 28
+           target: default
+           arch: x86
+           profile: Nexus 6
+           disable-animations: true
+           script: ./gradlew executeScreenshotTests
+           working-directory: ./android
+      
+      -  name: Publish artefacts
+         if: always()
+         uses: actions/upload-artifact@v2
+         with:
+          name: tests_results.html
+          path: /Users/runner/work/maas-components/maas-components/android/app/build/reports/shot/verification/index.html
```

Comment:
Is the html file enough? Having downloaded it, it does not contain the actual images and diffs 😞